changelog.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+
+## Unreleased
+
+### Refactor
+
+* [3659](https://github.com/zeta-chain/node/pull/3659) - add a check to stop posting inbound votes if the cctx has already been created
+
```

Comment:
no changelog needed as this is already released

Reply:
Added the changelog as this includes minor modifications to the tests, but I think it's fine; the changes are insignificant